src/cr/cube/dimension.py

```diff
@@ -629,33 +639,33 @@ def __init__(self, dimension_type, dimension_dict, dimension_transforms_dict):
 
     @lazyproperty
     def shimmed_dimension_dict(self) -> Dict:
-        """Copy of dimension dictionary with shimmed `element_id`s
+        """Augmented dimension dictionary with shimmed `element_id`s
 
         We want to move to a world where elements on a subvariables dimension are
         identified by their alias, but right now the "element_id" from zz9 is
         an index for subvariables.
         """
-        shim = copy.deepcopy(self._dimension_dict)
+        shim = self._dimension_dict
```

Comment:
Please check the other comment. My main concern is the same.

Reply:
same answer